api/v1/requests/TokenRequest.js

```diff
@@ -0,0 +1,20 @@
+var Request = require('./Request');
+
+var required = ['token', 'password'];
+var validations = {
+	'token': ['string'],
+	'password': ['string', 'minLength:8']
+};
+
+// usable whenever a request is made to obtain an token
```

Comment:
I'm thinking it doesn't hurt to have, especially since other developers can use this to help debug their new module. If you think it's too redundant I can remove it